javaslang/src/main/java/javaslang/collection/PriorityQueue.java

```diff
@@ -558,8 +571,8 @@ private Node(T root, int rank, Seq<Node<T>> children) {
                 assert rank == tree.rank;
 
                 return comparator.isLessOrEqual(this.root, tree.root)
-                       ? of(this.root, this.rank + 1, tree.appendTo(this.children))
-                       : of(tree.root, tree.rank + 1, this.appendTo(tree.children));
+                        ? of(this.root, this.rank + 1, tree.appendTo(this.children))
```

Comment:
It's the same ternary alignment I mentioned below.
It's an unrelated modification anyway (that doesn't make the code better), it's reason I recommended a revert.